### PR TITLE
supervisord: allow updating single process groups

### PR DESCRIPTION
**Pull request by HerrBerg**
supervisord allows to update single service groups.
This commit adds this feature while preserving the
old behavior when no service group is specified

### CODE REVIEW

**Comment by jtand** [comment]
@HerrBerg Thanks for this! It looks like there was a test failure, mind taking a look at it?

File: salt/modules/supervisord.py [hunk 1]
```diff
@@ -245,10 +245,13 @@ def reread(user=None, conf_file=None, bin_env=None):
     return _get_return(ret)
 
 
-def update(user=None, conf_file=None, bin_env=None):
+def update(name=None, user=None, conf_file=None, bin_env=None):
```
Comment (cachedout) [inline salt/modules/supervisord.py]: There may be people calling this positionally. It would be better to move the new kwarg to the end of the list.

**Comment by cachedout** [comment]
@HerrBerg Since I didn't hear back from you, I am going to merge this and then submit a follow-up PR to move the new kwarg to the end of the list of arguments. Thanks.

**Comment by HerrBerg** [comment]
That's fine with me. Thanks for merging.

**Pull request merged**